commands/link/README.md

```diff
@@ -10,7 +10,7 @@ Install [lerna](https://www.npmjs.com/package/lerna) for access to the `lerna` C
 $ lerna link
 ```
 
-Symlink together all Lerna `packages` that are dependencies of each other in the current Lerna repo.
+Symlink together all Lerna `packages` that are dependencies of each other in the current Lerna repo. When using `publishConfig.directory`, that will be the source directory of the link (e.g. `package-1/dist => node_modules/package-1`).
```

Comment:
You should document this separately, as it is done in the [publish README](https://github.com/lerna/lerna/tree/master/commands/publish#publishconfigdirectory). It should also be documented in the bootstrap README, if it is intended to be used there as well (I assume it is?). The `publishConfig.directory` namespace is lamentable for these cases, but it is what it is.

You'll also need to copy over the `--contents` [config](https://github.com/lerna/lerna/blob/afb04146796acf66611baefd06da4170ca1162b6/commands/publish/command.js#L27-L32) from the publish command into bootstrap and link (with appropriate modifications to the `description` field, obviously).